app/src/main/java/com/kickstarter/services/interceptors/GraphQLInterceptor.kt

```diff
@@ -5,20 +5,25 @@ import com.kickstarter.libs.Build
 import com.kickstarter.libs.CurrentUserType
 import com.kickstarter.libs.utils.WebUtils
 import okhttp3.Interceptor
+import okhttp3.Interceptor.Chain
 import okhttp3.Response
 
 class GraphQLInterceptor(private val clientId: String,
```

Comment:
Nit: Is it possible we could add documentation here since we're modifying this class already? Would help provide context as to what the purpose of this class is. Just an idea!

Reply:
Done!